lib/http/cookie_jar/yaml_saver.rb

```diff
@@ -73,4 +73,14 @@ def load(io, jar)
   def default_options
     {}
   end
+
+  if YAML.name == 'Psych' && Psych::VERSION >= '3.1'
```

Comment:
"3.10" > "3.1", in case you wonder.

Reply:
I am so very late here (thanks for doing this update btw), but in the distant future "10.0" < "3.1" :)